package/yast2-cluster.spec

```diff
@@ -12,67 +12,64 @@
 # license that conforms to the Open Source Definition (Version 1.9)
 # published by the Open Source Initiative.
 
-# Please submit bugfixes or comments via http://bugs.opensuse.org/
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
+%define _fwdefdir %{_libexecdir}/firewalld/services
 
 Name:           yast2-cluster
-Version:        4.1.2
+Version:        4.2.0
 Release:        0
+Summary:        Configuration of cluster
+License:        GPL-2.0-only
+Group:          System/YaST
+Url:            https://github.com/yast/yast-cluster
 
-BuildArch:      noarch
-BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 Source0:        %{name}-%{version}.tar.bz2
 Source1:        cluster.firewalld.xml
 
 # Yast2::Systemd::Socket
 BuildRequires:  yast2 >= 4.1.3
-Requires:       yast2 >= 4.1.3
-
 BuildRequires:  perl-XML-Writer
 BuildRequires:  update-desktop-files
 BuildRequires:  yast2-devtools >= 3.1.10
```

Comment:
The `%yast_metainfo` macro has been added in `yast2-devtools-4.2.2`, please update the dependency.